lib/src/base-styles/_nx-variables.scss

```diff
@@ -65,9 +65,14 @@
                             0 1px 2px 0 rgba(91, 95, 141, 0.2),
                             0 1px 1px 0 rgba(124, 128, 179, 0.2);
 
+  // NxDrawer
+  --nx-box-shadow-drawer: -34px 0 50px -32px hsla(0, 0%, 50%, 0.5);
```

Comment:
Do we want to parameterize this whole thing or just the color?  The positioning is pretty specific to `NxDrawer` if I'm not mistaken

Reply:
> Do we want to parameterize this whole thing or just the color? The positioning is pretty specific to `NxDrawer` if I'm not mistaken

This is just for drawer, so I think you're right to just parameterize the color. Done so here: [7e25ef6](https://github.com/sonatype/sonatype-react-shared-components/pull/1097/commits/7e25ef66b14f6de8c1d93ad316da5864a951b0cb)